pipeline/reference/__init__ 2.py

```diff
@@ -0,0 +1,4 @@
+
+from .ccf_ontology import ccf_ontology
+
+__all__ = [ccf_ontology]
```

Comment:
Where is this file coming from? I don't think it's needed here (if not, please delete)

Reply:
Will do.